lang/cs/Org.Apache.REEF.Common/Telemetry/MetricTracker.cs

```diff
@@ -46,12 +45,8 @@ public sealed class MetricTracker : ITracker
         /// </summary>
         [JsonProperty]
         private ConcurrentQueue<MetricRecord> Records;
-        
-        /// <summary>
-        /// Number of times metric has been updated since last processed.
-        /// </summary>
-        [JsonProperty]
-        internal int ChangesSinceLastSink;
+
+        private IMetric Metric;
```

Comment:
naming conventions: use `_metric` for private members

Reply:
fixed.